common/cluster-api/src/main/proto/queue.proto

```diff
@@ -1040,15 +1153,18 @@ message ToCoreNotificationMsg {
   LocalSubscriptionServiceMsgProto toLocalSubscriptionServiceMsg = 1;
   FromDeviceRPCResponseProto fromDeviceRpcResponse = 2;
   bytes componentLifecycleMsg = 3; //will be removed in 3.6.1 in favour of ComponentLifecycleMsgProto
```

Comment:
add please `[deprecated = true]` for `bytes componentLifecycleMsg = 3` on by behalf, please . TY

Reply:
Ok, I'll do the same for ToRuleEngineNotificationMsg and for PE integrationNotificationMsg